keyboards/ploopyco/nano_2/readme.md

```diff
@@ -0,0 +1,34 @@
+# Ploopy Nano 2 Trackball
+
+![Ploopyco Trackball Nano](https://ploopy.co/wp-content/uploads/2021/06/2.jpg)
+
+It's a DIY, QMK Powered Trackball...Nano!
+
+* Maintainer: [PloopyCo](https://github.com/ploopyco)
+* Key contributors: [Drashna Jael're](https://github.com/drashna/), [Germ](https://github.com/germ/)
+* Hardware Supported: RP2040
+* Hardware Availability: [Store](https://ploopy.co/nano-2), [GitHub](https://github.com/ploopyco)
+
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile -kb ploopyco/nano_2 -km default
```

Comment:
I think this needs to be added to `data/keyboard_aliases.hjson` for this to work:
```
"ploopyco/nano_2": {
        "target": "ploopyco/nano_2/rev2_003"
    }
```